src/index.ts

```diff
@@ -37,17 +37,18 @@ const parseDate = (d: string): number => {
 	const m = /Date\((.*)\)/.exec(d);
 	return m ? parseInt(m[1], 10) : 0;
 };
+let isOutsideUs = false;
```

Comment:
Can we make it such that `isOutsideUs` is not a global variable? It should be possible to have two separate instances of the iterator, where one is inside the US and another is outside the US.

Reply:
Sure, that's a valid point. It should be fixed now but unfortunately I'm currently waiting for my health insurance to send me my first regular sensor and can't test it (I was using one from a clinical trial before). The request seems to be valid and doesn't raise any errors but I currently don't get any readings back either - probably because I haven't shared any glucose readings in a few days.